raiden/routing.py

```diff
@@ -132,7 +98,34 @@ def get_best_routes(
             )
             continue
 
-        route_state = RouteState(partner_address, channel_state.identifier)
-        available_routes.append(route_state)
+        nonrefundable = amount > channel.get_distributable(
```

Comment:
this is the same check as in line 80, so if the condition is true, it will have skipped this iteration already.

Reply:
also, there's already a variable for `distributable`

Reply:
Arguments of `get_distributable()` have been switched , so it should check the backwards route capacity.